changelog/changelog.py

```diff
@@ -307,15 +316,17 @@ def main():
         'generator': default_changelog_generator,
     },
     'spinalcordtoolbox': {
-        'labels': ['feature', 'documentation-internal', 'CI', 'bug', 'installation', 'documentation', 'enhancement', 'refactoring', 'git/github'],
+        'labels': ['feature', 'documentation-internal', 'CI', 'bug', 'installation', 'documentation', 'enhancement',
+                   'refactoring', 'git/github'],
         'generator': sct_changelog_generator,
     },
     'ivadomed': {
-        'labels': ['bug', 'dependencies', 'documentation', 'enhancement'],
+        'labels': ['feature', 'CI', 'bug', 'installation', 'documentation', 'dependencies', 'enhancement',
```

Comment:
do we need to separate CI and testing? i guess that's more a question for the ivadomed team...

Reply:
I agree with @jcohenadad, `CI` and `testing` are kind of redundant. I would keep only `CI`.

Also in `ivadomed`, please make sure that the new "dark purple" labels descriptions start with "`category: `", this is use for easy filtering or the "PR" categories in the documentation. Thanks!

Reply:
So the rationale with `CI` tag vs `testing` tag was that `CI` tags leans heavily related to `GitHub Action` related continuous integration modification which are not always guaranteed to be `testing` related. For instance. We have CI scripts that does quite a few additional things beside testing. 

![image](https://user-images.githubusercontent.com/6720068/131601696-a0cf0545-83a0-4263-9840-3207bf3329d8.png)

`CI` tags PRs almost can never be tested completely locally and only executable on `GitHub Actions` (not even other CI platforms)

As for the `changelog` audience/end consumer (such as IvadoMed end users), `CI` tag related information might not be super useful and we might be able to skip `CI` tags as those typically are not useful/relevant to the user.

Reply:
On second thought, more conservatively, I think it might be better to at least keep both for now and carry out a manual review of the `CI` vs `Test` tagged PR titles in the actual changelog output text file as I am not 100% sure the PR tagging of devs are done with this subtlety in mind so better safe than sorry (and miss out on PRs). Then based on what I find in the output for v2.8.0 release , we can adjust it in another PR after ensuring alignment across IvadoMed devs on consistent tagging.